### PR TITLE
Updated agenda-view according to latest test style

### PR DESCRIPTION
**Pull request by jloosli**
@arshaw I tweaked one file to just make sure I was doing it correctly. Please review and let me know if there are things I need to change.

### CODE REVIEW

**Comment by jloosli** [comment]
Also, based on the example of Agenda-render.js, jQuery was still left in the tests. Are you wanting to convert to non-jquery dom lookups as this time also, or are you wanting to hold off on that for this first pass?

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ dist
 
 # npm
 node_modules
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+.idea
```
Comment (arshaw) [inline .gitignore]: could you remove this? leave the .gitignore untouched

**Comment by arshaw** [comment]
hi @jloosli , thanks so much for your help! this looks great.

you can just keep jquery within the tests. the only thing we care about is getting `$().fullCalendar` calls out of there.

i might eventually look for a jasmine plugin that makes this stuff easier, similar to what jasmine-jquery does, for for raw DOM nodes.

**Comment by jloosli** [comment]
@arshaw OK...fixed .gitignore and reverted back to the jQuery version of the test

**Comment by arshaw** [comment]
this might have been auto-closed because of a merge conflict with code i just pushed? i'm not sure

**Comment by arshaw** [comment]
i found a way to get this work into master: https://github.com/fullcalendar/fullcalendar/commit/6924c42fc364aaf7416561876d06441d136e1915

**Pull request closed without merging**